waffle_utils/dataset/dataset.py

```diff
@@ -256,7 +258,87 @@ def from_coco(
             )
 
         # copy raw images
-        io.copy_files_to_directory(coco_root_dir, ds.raw_image_dir)
+        io.copy_files_to_directory(images_dir, ds.raw_image_dir)
+
+        return ds
+
+    @classmethod
+    def from_yolo(
```

Comment:
this function only supports object_detection. check the classification format.

Reply:
I think we should add `task` in `Dataset` class.
😂😂😂😂

Reply:
What you mean is, separating how the `from_yolo()` imports the dataset according to the task? Such as like this?
```python
# waffle_utils/dataset/dataset.py
class Dataset:
    ...
    def get_task(...):
        ...
       return task

    def from_yolo(...):
        task = get_task()
        if task == 'classification':
            ...
        elif task == 'object_detection':
            ...
        else:
            ...
```

Reply:
Ah, into the property. now I get it.